### PR TITLE
Make PortStatus tests use the available raw dump

### PR DESCRIPTION
**Pull request by renanrodrigo**

### CODE REVIEW

File: pyof/v0x01/common/phy_port.py [hunk 1]
```diff
@@ -153,9 +153,9 @@ class PhyPort(GenericStruct):
     supported = UBInt32(enum_ref=PortFeatures)
     peer = UBInt32(enum_ref=PortFeatures)
 
-    def __init__(self, port_no=None, hw_addr=None, name=None, config=None,
-                 state=None, curr=None, advertised=None, supported=None,
-                 peer=None):
+    def __init__(self, port_no=None, hw_addr=None, name=None, config=0,
+                 state=PortState.OFPPS_STP_LISTEN, curr=0, advertised=0,
+                 supported=0, peer=0):
```
Comment (diraol) [inline pyof/v0x01/common/phy_port.py]: @renanrodrigo where does this values come from? Why you have choosed them as default? [just trying to understand]

Comment (renanrodrigo) [inline pyof/v0x01/common/phy_port.py]: They are not mandatory, but were chosen based on the OpenFlow specification.

`config` is a 'flags' field. The flags are defined in PortConfig, but if no flag is set (default for the OVS switch that generated the dump we have) the value is zero.

`state` is the STP state of the physical port - the default value is LISTEN.

The four features fields have zero as value when they are unavailable or unsupported by the switch - it happens to three of them in the OVS dump.

Comment (diraol) [inline pyof/v0x01/common/phy_port.py]: Great, tks!

Comment (cemsbr) [inline pyof/v0x01/common/phy_port.py]: Nice. We need sensible struct defaults everywhere.

**Comment by beraldoleal** [comment]
@renanrodrigo I'm going to accept, but please try to split PRs on future PRs.

**Pull request merged**